src/components/Tailwind/Cards/EpochReleaseTimerCard.tsx

```diff
@@ -58,15 +60,16 @@ const EpochReleaseTimerCard = ({ event, content, countdown }: TimeObject) => {
           </div>
           <img className="h-155.61 hidden sm:block" alt="" src={EpochCharacter} />
         </div>
-        <div className="flex flex-col md:flex-row text-center md:text-left mx-4 md:space-x-8 space-y-2 md:space-y-0 mt-4 ">
+        <div className="flex flex-col md:flex-row text-center md:text-left mx-4 space-y-2 md:space-y-0 mt-4 justify-between">
           <div className="flex flex-col">
             <p className="text-12px bold ">DEPOSIT IN POOL </p>
             <h4 className="text-lg">{content.deposit}</h4>
           </div>
-          <div className="flex flex-col">
-            {/*<p className="text-12px bold">MONTHLY PROFITS </p>*/}
-            {/*<h4 className="text-lg">{content.profit}</h4>*/}
-          </div>
+          {chainId !== 1 && (
```

Comment:
Better to use the `ChainId` enum when comparing

```suggestion
          {chainId !== ChainId.MAINNET && (
```

Reply:
@schystz done.